src/metalnx-web/src/main/java/com/emc/metalnx/controller/MetadataController.java

```diff
@@ -91,6 +94,12 @@ public class MetadataController {
 	@Value("${irods.zoneName}")
 	private String zoneName;
 
+	@Value("${metalnx.metadata.mask.prefixes}")
```

Comment:
singular or plural... i suppose plural is more correct...

Reply:
done